roles/load-balancers/manage-haproxy/templates/lb_frontend.j2

```diff
@@ -1,54 +1,51 @@
-
-{% if ((fe.lb_mode is defined and fe.lb_mode == 'http') or
-       (fe.lb_mode is undefined and fe.lb_host_port|string == '80')) %}
-{% set lb_http_mode = True %}
-{% else %}
-{% set lb_http_mode = False %}
-{% endif %}
+{% set lb_default = namespace(needed=True) -%}
+{% set lb_ssl = namespace(enabled=False) -%}
 
 {% if ((fe.lb_ssl_enabled is defined and fe.lb_ssl_enabled) or
        (fe.lb_ssl_enabled is undefined and fe.lb_host_port|string == '443')) %}
 {% set lb_ssl_enabled = True %}
```

Comment:
Should this be `{% set lb_ssl.enabled = True %}`

Reply:
yes - fixed. thx for catching